inst/shiny/app.R

```diff
@@ -43,6 +43,12 @@ ui <- function() {
     id = "sidebar",
     title = "aNCA",
 
+    div(
+      class = "project-name-container",
+      textInput("project_name", label = NULL, placeholder = "Project Name"),
+      icon("file", class = "project-name-icon")
+    ),
```

Comment:
suggestion: My proposition would be to add the project name input to the top / title bar, I think it looks nice.
```suggestion
    title = tagList(
      span("aNCA"),
      div(
        class = "project-name-container",
        textInput("project_name", label = NULL, placeholder = "Project Name"),
        icon("file", class = "project-name-icon")
      )
    ),
```